butlarr/services/ext.py

```diff
@@ -112,3 +114,78 @@ async def cmd_help(self, update, context, args):
             response_message += f"\n - `/{self.commands[0]} {cmd} {escape_markdownv2_chars(pattern)}` \t _{escape_markdownv2_chars(desc)}_"
 
         return await update.message.reply_text(response_message, parse_mode="Markdown")
+    
+    def media_caption(self, item, overview=True):
```

Comment:
The implementation of this should be moved to ArrService imo.
I would even argue that we should force users to implement it by throwing a NotImplemented Exception per default, but that's more a taste thing.
But this for sure shouldn't be part of the ExtService :)

Reply:
Changed it too ArrService, but left the implementation there since the behavior should be the same for sonarr and radarr, if any service needs another behavior then it can be reimplemented there